packages/website/gatsby-config.js

```diff
@@ -6,7 +6,7 @@ module.exports = {
     description: `Skynet is a decentralized file sharing and content distribution protocol`,
     author: `Skynet Labs`,
     siteUrl: `https://siasky.net`,
```

Comment:
Should this be configurable as well?

Reply:
well yeah but let me make some changes then

Reply:
I had to switch back from building static files in image and service them to building locally to use env vars to substitute the site url.